docs/source/tutorials/05_calling_peekingduck_in_python.rst

```diff
@@ -239,23 +238,25 @@ Lines 17 - 23: The debugging output showing the frame number and the confidence
 bounding boxes predicted as "cat".
 
 
-Running in a Notebook
-=====================
+Integrating with Your Workflow
+==============================
 
 The modular design of PeekingDuck allows users to pick and choose the nodes they want to use. Users
-are also able to use PeekingDuck nodes with external libraries when designing their pipeline.
+are also able to use PeekingDuck nodes with external packages when designing their pipeline.
 
 In this demo, we will show how users can construct a custom PeekingDuck pipeline using:
 
     * Data loaders such as `tf.keras.utils.image_dataset_from_directory
       <https://www.tensorflow.org/api_docs/python/tf/keras/utils/image_dataset_from_directory>`_
       (available in ``tensorflow>=2.3.0``),
-    * External models (not implemented as PeekingDuck nodes) such as `easyocr
+    * External packages (not implemented as PeekingDuck nodes) such as `easyocr
       <https://pypi.org/project/easyocr/>`_, and
     * Visualization packages such as `matplotlib <https://pypi.org/project/matplotlib/>`_.
 
-The notebook corresponding to this tutorial can be found in the `notebooks <https://github.com/aimakerspace/PeekingDuck/tree/dev/notebooks>`_ folder of the
-PeekingDuck repository and is also available at a `Colab notebook <https://colab.research.google.com/drive/1NwQKrnY_3ia2mBEaUinkvUqbrjjT3ssq#scrollTo=l2MCyh5Hgp5O>`_.
+The notebook corresponding to this tutorial, ``demo_import_peekingduck.ipynb``, can be found in the
```

Comment:
For consistency, can we rename this notebook and its heading?

Reply:
updated to `calling_peekingduck_in_python.ipynb`